net-misc/ncp/metadata.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>bu9zilla@gmail.com</email>
+		<name>Michael Mair-Keimberger</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 <longdescription lang="en">
```

Comment:
Let's indent this as well, just like above.

Reply:
Done, also indented the ```<use><flag...```